harbour-advanced-camera.pro

```diff
@@ -68,4 +69,8 @@ HEADERS += \
     src/wbmodel.h \
     src/focusmodel.h \
     src/flashmodel.h \
-    src/fsoperations.h
+    src/fsoperations.h \
+    src/resourcehandler.h
+
+INCLUDEPATH += /usr/include/resource/qt5
```

Comment:
Do we need to add anything to the .yaml/.spec?

Reply:
I think we should use pkg-config in both the make `libresourceqt5` a dependency. Same for this .pro file should use pkg-config to set needed things `CONFIG += link_pkgconfig
PKGCONFIG += libresourceqt5`. You can check https://github.com/nemomobile/qmlgallery how each part is done.

Reply:
Using libresourceqt5 is not possible since it is not allowed in the harbour.